r/PACKAGING.md

```diff
@@ -0,0 +1,117 @@
+
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# Packaging checklist for CRAN release
+
+For full documentation of the steps in this checklist, see the
+[Apache Arrow Release Management Guide](https://arrow.apache.org/docs/developers/release.html#post-release-tasks).
+
+- [ ] When a release candidate is created
+  [Create a GitHub issue](https://github.com/apache/arrow/issues/new/)
+  entitled `[R] CRAN release version X.X.X` and copy this checklist to the issue
+  
+- [ ] Confirm that all
+ R-related [nightly tests and nightly packaging builds](https://lists.apache.org/list.html?builds@arrow.apache.org) from immediately before the RC was cut are passing.
+- [ ] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_arrow.html)
```

Comment:
```suggestion
- [ ] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_arrow.html) and ensure any ERRORs and WARNINGs from the last release have been fixed
```
Perhaps this? I'm confused, I thought things wouldn't have made it onto CRAN with ERROR/WARNING things in there?

Reply:
Aha, I see!  So, in this case the ERRORs are from the `lubridate::qy()` update, so we're good for this one.  Thanks for the context there @assignUser